### PR TITLE
Added Laravel example

### PR DESCRIPTION
**Pull request by aios**
Example for Laravel Framework

### CODE REVIEW

File: laravel/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+example.com {
+
+    root ./public
+    log ./storage/logs/caddy-access.log
+    errors ./storage/logs/caddy-error.log
```
Comment (mholt) [inline laravel/Caddyfile]: This example can probably be simplified without `log` and `errors`. Also, is `root` necessary (i.e. is it common for laravel apps to be in a different folder called "public")?

File: laravel/Caddyfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    fastcgi / 127.0.0.1:9000 php {
+        index index.php
```
Comment (mholt) [inline laravel/Caddyfile]: Hmm I don't think `index` is necessary, since `index.php` is the default.

File: laravel/Caddyfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    }
+    
+    rewrite {
+        r .*
```
Comment (mholt) [inline laravel/Caddyfile]: I think we can drop this line. Right, @abiosoft?

File: laravel/Caddyfile (hunk 1 above)
Comment (twoSeats) [inline laravel/Caddyfile]: @mholt I agree that `log` & `errors` are not required. All laravel apps default to "public" as the root/"public" folder though.

Comment (mholt) [inline laravel/Caddyfile]: Okay cool. Let's remove the log and errors lines but keep the root line.

File: laravel/Caddyfile (hunk 3 above)
Comment (mholt) [inline laravel/Caddyfile]: Update: Abiola suggested a more concise rewrite in the forum; maybe use that, if it works!

**Comment by twoSeats** [comment]
Improving on @mholt's comments & @abiosoft's input in the [forum](https://forum.caddyserver.com/t/laravel-caddyfile-example/520).
The solution could be simplified to the following I think.
```
example.com {
    root ./public
    fastcgi / 127.0.0.1:9000 php
    rewrite {
        to {path} {path}/ /index.php?{query}
    }
} 
```

**Comment by aios** [comment]
added...

**Comment by mholt** [comment]
Thanks!

**Comment by aios** [comment]
Np)

**Pull request merged**